packages/backend-for-frontend/src/services/catalogService.ts

```diff
@@ -1444,6 +1454,25 @@ export function catalogServiceBuilder(
         isClientAccessDelegable: importedEservice.isClientAccessDelegable,
       };
 
+      const eservice: catalogApi.EService = importedEservice.templateRef
+        ? await catalogProcessClient.createEServiceInstanceFromTemplate(
+            {
+              instanceId: importedEservice.templateRef.instanceId,
```

Comment:
Update the base branch, this should now be instanceLabel
```suggestion
              instanceLabel: importedEservice.templateRef.instanceLabel,
```